test/handlers/kubectl/K8sKubectlBaseTestSuite.ts

```diff
@@ -0,0 +1,10 @@
+import {K8sBaseHandlerTestSuite} from '../K8sBaseHandlerTestSuite';
+import {Container} from 'typedi';
+import {TempPathsRegistry} from 'fbl/dist/src/services';
+
+export class K8sKubectlBaseTestSuite extends K8sBaseHandlerTestSuite  {
```

Comment:
Based on our verbal discussion:
- Remove this class.
- Update `K8sBaseHandlerTestSuite` after method to not be `static`
- Change all classes that were extended from `K8sKubectlBaseTestSuite` to be now extended from `K8sBaseHandlerTestSuite`